.gitignore

```diff
@@ -90,6 +90,7 @@ celerybeat-schedule
 .env
 
 # virtualenv
+.venv/
```

Comment:
driveby

Reply:
Typically this kind of stuff should be in your [global gitignore](http://egorsmirnov.me/2015/05/04/global-gitignore-file.html). But I'm not against adding it here.

Reply:
Oh is there a non-global .gitignore?!

Reply:
The `.gitignore` is per repo. You can also create a global file which applies to all projects on your computer. Follow the link I shared and check it out.

Anything which is a personal preference like your editor (`.vscode`, `.idea`, etc), environment manager, etc should probably go in your global gitignore file.

Reply:
Oh nice, I didn't even realize that was a thing. But it totally makes sense. (I was reading on my phone before and didn't even notice the link.)